.github/workflows/build.yaml

```diff
@@ -56,6 +56,8 @@ jobs:
           path: ~/go/bin/rq
           key: ${{ runner.os }}-${{ runner.arch }}-go-rq-${{ env.RQ_VERSION }}
       - run: build/do.rq pull_request
+      - run: go test -race ./...
+        if: matrix.os.name == 'linux'
```

Comment:
I guess this is tolerable since the intel macos build is going to be slower anyway 😄 But when we get rid of that, we should run this in a parallel job.